app/src/main/java/it/bz/beacon/adminapp/swagger/client/JSON.java

```diff
@@ -180,6 +180,8 @@ public OffsetDateTime read(JsonReader in) throws IOException {
                     if (date.endsWith("+0000")) {
                         date = date.substring(0, date.length()-5) + "Z";
                     }
+                    System.out.println(formatter.toFormat().toString());
```

Comment:
@noctho Do you want to keep these lines?

Reply:
No, I don't want to keep the lines. I have regenerated the swagger client classes and they are no longer there.